packages/vite/bin/vite.js

```diff
@@ -10,8 +10,14 @@ if (!__dirname.includes('node_modules')) {
 global.__vite_start_time = Date.now()
 
 // check debug mode first before requiring the CLI.
-const debugIndex = process.argv.indexOf('--debug')
-const filterIndex = process.argv.indexOf('--filter')
+const debugIndex =
+  process.argv.indexOf('--debug') > 0
+    ? process.argv.indexOf('--debug')
+    : process.argv.indexOf('-d')
```

Comment:
How about use check `--d` instead of `--debug`? It can inlcude two case.

Reply:
That's a much better idea, didn't think of that.

Reply:
Though, actually, wouldn't that require looping through all of `process.argv`? `indexOf` is only going to return exact matches, as far as I know.

Reply:
What about this?
```js
const debugIndex = process.argv.findIndex(arg => /^(?:-d|--debug)$/.test(arg))
```